recipe/meta.yaml

```diff
@@ -13,13 +13,15 @@ build:
   number: 1
```

Comment:
The build number should be incremented as well.

Reply:
Yes, I'll update.